dashboard/src/components/ThemeSelector/DarkTheme.scss

```diff
@@ -0,0 +1,16 @@
+// CSS tweaks the dark mode theme theme
+
+.section-not-found img {
+  // Convert to white
+  filter: invert(1) brightness(100%) saturate(0%);
+}
+
+.kubeapps-header-subtitle img {
+  // Convert to white
+  filter: invert(1) brightness(100%) saturate(0%);
+}
+
+// Convert to white
+.bg-img img {
+  filter: invert(1) brightness(100%) saturate(0%);
```

Comment:
I was just reading about scss to dry this up while still having clear, separate selectors. Not important at all, but see we can do something like:

```
%convert-to-white {
  filter: invert(1) brightness(100%) saturate(0%);
}
.section-not-found img {
  @extend %convert-to-white;
}
...
```

Reply:
Thanks a lot for the tip. I haven't extensively used scss and wasn't aware of this `@extend`. Really useful :)